### ISSUE TITLE
Highlight tiles to which you move

### ISSUE THREAD

**Issue by sorskoot** [sorskoot/js13kgames_2017_Lost#1]
When 'hovering' over a tile there should be some indication that that tile is where the player is going to move to when selected.

**Comment by sorskoot** [comment]
It would be nice to have a better shader that shows an animating border around the tile you selected.

**Comment by sorskoot** [comment]
I think #28 should be done first.
